cura/CrashHandler.py

```diff
@@ -9,11 +9,11 @@
 import os.path
 import time
 import json
-import ssl
-import urllib.request
-import urllib.error
+from typing import cast
 
-import certifi
+from sentry_sdk.hub import Hub
+from sentry_sdk.utils import event_from_exception
+from sentry_sdk import configure_scope
```

Comment:
we should make sure Lipu's PR (add `sentry_sdk`) ends up in the build env. :-)

Reply:
He didn't do it with a PR, so it's already in the build env.

Reply:
Nice.